go/state/state_db.go

```diff
@@ -823,6 +844,36 @@ func (s *stateDB) SetState(addr common.Address, key common.Key, value common.Val
 	}
 }
 
+func (s *stateDB) GetTransientState(addr common.Address, key common.Key) common.Value {
+	sid := slotId{addr, key}
+	val, _ := s.transientData.Get(sid)
+	return val
+}
+
+func (s *stateDB) SetTransientState(addr common.Address, key common.Key, value common.Value) {
+	if s.createAccountIfNotExists(addr) {
```

Comment:
I think this operation should not create an account.

Reply:
So the value should not be stored, if the account does not exist?

Reply:
store the value, but do not create the account.

not sure if it can happen that the value is set for non-existing account, but it is how it works in ethereum
https://github.com/ethereum/go-ethereum/blob/master/core/state/statedb.go#L522-L539